compose-builder/res/devices/openvino.toml

```diff
@@ -0,0 +1,9 @@
+# Pre-define Devices
```

Comment:
where is the file being used?

Reply:
File was not meant to be included. Will remove.